src/algolia-projects.json

```diff
@@ -76,6 +76,19 @@
     "ranking": 1500
   },
   {
+    "name": "Vue InstantSearch",
+    "category": "Library",
+    "description": "The Vue.js ultimate toolbox for creating instant-search experiences",
+    "url_home": "https://community.algolia.com/vue-instantsearch/",
+    "url_github": "https://github.com/algolia/vue-instantsearch",
+    "url_forum": "",
+    "discussion_count": "",
+    "icon": "instantsearch-react",
```

Comment:
Can you change icon : "vue-instantsearch" or "instantsearch-vue" and I'll add the icon :)?